app/controller/auth.controller.js

```diff
@@ -0,0 +1,127 @@
+const passport = require('passport');
```

Comment:
Codacy has a fix for the issue: [Replace `'passport'` with `"passport"`](https://app.codacy.com/gh/Eight-Corner/oneuleun_web_api/pullRequest?prid=9684192)
```suggestion
const passport = require("passport");
```